docs/reference/api/pages.md

```diff
@@ -0,0 +1,511 @@
+
+The `_position` property uses specific string values rather than index numbers to better support the draft review workflow.
+
+### Request example
+
+```javascript
+// Object with, at a minimum, properties for each required piece field.
+const data = {
+  title: 'My great new article',
+  // Content properties, e.g., `category`, `mainContent`
+  _targetId: 'ckhdscx5900054z9k88uqs16w',
+  _position: 'lastChild'
+};
+// Request inside an async function.
+const response = await fetch('http://example.net/api/v1/@apostrophecms/page/ckitdo5oq004pu69kr6oxo6fr?apikey=myapikey', {
+  method: 'PUT',
+  headers: {
+    'Content-Type': 'application/json'
+  },
+  body: JSON.stringify(data)
+});
+```
+
+### Response
+
+The successful PUT request returns the newly created document. See the [page document response example](#page-document-response-example) below for a sample response body.
+
+## `PATCH: /api/v1/@apostrophecms/page/:id`
+
+**Authentication required.**
+
+The PATCH request may include *both* `_targetId` and `_position` as described in the [POST request description](#post-api-v1-apostrophecms-page), but that is not required if the page is not being moved.
+
+### Request example
+
+```javascript
+// Object with *only* the document fields to overwrite.
+// This example only changes the page's title to "Organization history."
+const data = {
+  title: 'Organization history'
+};
+// Request inside an async function.
+const response = await fetch('http://example.net/api/v1/@apostrophecms/page/ckitdo5oq004pu69kr6oxo6fr?apikey=myapikey', {
+  method: 'PATCH',
+  headers: {
+    'Content-Type': 'application/json'
+  },
+  body: JSON.stringify(data)
+});
+```
+
+:::tip
+As a convenience, you may make a PATCH request for any MongoDB document, regardless of type using a catch-all route using the document's `_id` property:
+```
+PATCH /api/v1/@apostrophecms/doc/:id
+```
+:::
+
+### MongoDB-style requests
+
+The PATCH request body may use MongoDB-style operators. For example, you may use dot or "at" notation to update a nested property:
+
+```json
+{
+  // Via "dot notation"
+  "description.items.0.content": "<p>Update only the rich text.</p>",
+  // Same thing via "@ notation," which finds the nested item with that _id
+  "@ckgwegpfw00033h5xqlfb74nk.content": "<p>Update only the rich text.</p>"
+}
+```
+
+### Response
+
+The successful PATCH request returns the newly created document. See the [page document response example](#page-document-response-example) below for a sample response body.
+
+### Moving pages to the trash
+
+The trash is part of the overall page tree in order to maintain the nesting structure. As such, there is not only a simple `trash` property to set `true`. Instead, set `_targetId` to `_trash` and `_position` to `lastChild` (or another position within the trash). You may similarly move pages out of the trash by moving them to a position relative to another page that is not in the trash.
```

Comment:
We should document both special convenience values, `_home` and `_trash`, each time `_targetId` is covered if we haven't already.